azure-functions/acbs-function/mappings/facility/facility-covenant-amend.js

```diff
@@ -0,0 +1,25 @@
+import { to2Decimals } from '../../helpers/currency';
+
+export const facilityCovenantAmend = (amendment) => {
+  try {
+    const { amount, facilityGuaranteeDates } = amendment;
+
+    const mappedResponse = {};
+    if (!amount && !facilityGuaranteeDates) {
+      throw new Error('Either amount or facilityGuaranteeDates.guaranteeExpiryDate is required.');
+    }
+
+    if (amount) {
+      mappedResponse.targetAmount = to2Decimals(amount);
```

Comment:
@abhi-markan are we expecting some kind of conversion here, I assume not?

Reply:
Why do we want to assign in this pattern please, as it makes it unextendable from any line of code above this?